api/src/main/java/org/pmiops/workbench/billing/BillingProjectBufferService.java

```diff
@@ -138,11 +144,15 @@ private String createBillingProjectName() {
     return prefix + randomString;
   }
 
+  private int getRemainingBufferSlotsSize() {
+    return getBufferMaxCapacity() - (int) getCurrentBufferSize();
```

Comment:
I'd expect `getCurrentBufferSize` to return the number of projects in the buffer right now, not the number of unfilled slots. I understand its an existing thing, but maybe `getCurrentBufferSpace` (or `getCurrentUnfilledBufferSpace`?)

Reply:
that is what `getCurrentBufferSize()` returns. I really like `getCurrentUnfilledBufferSpace` over `getRemainingBufferSlotsSize`. will make the switch!